Update panel XSLT for LogixNG Digital Actions, Expressions, Boolean Actions

jerryg2003 opened the pull request:
Last update in March 2022.  May need additions for other newer LogixNG features but wanted to get these as a start.

danielb987 commented:
@jerryg2003 
Thank you!

@dsand47 
Can you take a look at this? It looks good to me, but I know very little about XSLT.

jerryg2003 commented:
@dsand47 Thanks.  Have incorporated suggestion and made additional enhancements to LogixNG display.  Have tested with many panels but may still be missing some detail items I am not aware of.  Happy to add more if any of you have more complex LogixNG panels I can test with.  @danielb987

dsand47 commented on xml/XSLT/panelfile-4-19-2.xsl:
```diff
@@ -15,7 +15,7 @@
 -->
 
-<!-- This file is part of JMRI.  Copyright 2007-2011, 2016, 2018, 2022, 2023.     -->
+<!-- This file is part of JMRI.  Copyright 2007-2011, 2016, 2018, 2022.     -->
```
Copyright year lost.

jerryg2003 replied:
Fixed.

dsand47 commented on xml/XSLT/panelfile-4-19-2.xsl:
```diff
@@ -24,7 +24,7 @@
      because XPath will evaluate '1997 - 2017' to '20'.
 -->
-<xsl:param name="JmriCopyrightYear" select="concat('1997','-','2023')" />
+<xsl:param name="JmriCopyrightYear" select="concat('1997','-','2022')" />
```
Wrong copyright year.

danielb987 commented:
@jerryg2003 
The problems with copyright might be that you have created this PR from an old master branch. Try to merge upstream/master into this branch. That will also solve the merge conflict with current-draft-note.shtml.

jerryg2003 commented:
My bad on copyright year: stepped on my own version (now fixed).  I can't seem to figure out how to resolve the draft release notes conflict:  it was OK when I started this PR as I started from the master.  I then added one line but somehow that got out of sync with the master.  I updated just now to be the same as master with my line added but still says I have a conflict.  Please advise.  Thanks.

danielb987 commented:
I can fix it for you tomorrow.
It happened because of another PR was merged.

12 jan. 2023 01:46:07 JerryG ***@***.***>:

> My bad on copyright year: stepped on my own version (now fixed). I can't seem to figure out how to resolve the draft release notes conflict: it was OK when I started this PR as I started from the master. I then added one line but somehow that got out of sync with the master. I updated just now to be the same as master with my line added but still says I have a conflict. Please advise. Thanks.
> 
> —
> Reply to this email directly, view it on GitHub[https://github.com/JMRI/JMRI/pull/11724#issuecomment-1379665561], or unsubscribe[https://github.com/notifications/unsubscribe-auth/AE2REVJNVCD75FESSC3TA7LWR5H43ANCNFSM6AAAAAATULKB3Y].
> You are receiving this because you were mentioned.[Spårningsbild][https://github.com/notifications/beacon/AE2REVPBVZV6VQYTMJZEEIDWR5H43A5CNFSM6AAAAAATULKB32WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTSSHQDJS.gif]
>

dsand47 commented:
The following is a missing signal mast type.

```
    <olcbsignalmast class="jmri.jmrix.openlcb.configurexml.OlcbSignalMastXml">
      <systemName>MF$olm:MilwHD:SL-1($0005)</systemName>
      <userName>Alpha-Right-B</userName>
      <lit>
        <lit>03.00.02.23.03.30.15.39</lit>
        <notlit>03.00.02.23.03.30.15.3A</notlit>
      </lit>
      <held>
        <held>03.00.02.23.03.30.15.37</held>
        <notheld>03.00.02.23.03.30.15.38</notheld>
      </held>
      <aspect defines="Clear">
        <event>03.00.02.23.03.30.15.31</event>
      </aspect>
      <aspect defines="Approach">
        <event>03.00.02.23.03.30.15.32</event>
      </aspect>
      <aspect defines="Restricting">
        <event>03.00.02.23.03.30.15.35</event>
      </aspect>
      <aspect defines="Stop">
        <event>03.00.02.23.03.30.15.36</event>
      </aspect>
    </olcbsignalmast>
```

jerryg2003 commented:
@dsand47 can we let this PR go in as is since it is focused on LogixNG? I will then do a separate PR likely next week when I have a chance to figure out an appropriate  formatting  for OLCB signal masts. Thanks.

dsand47 commented:
> @dsand47 can we let this PR go in as is since it is focused on LogixNG formatting? I will then do a separate PR likely next week when I have a chance to figure out an appropriate  formatting  for OLCB signal masts. Thanks. 

No problem doing this PR first.
